### PR TITLE
Fix: use system time instead of UTC in ResourceManager.share

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: golem/resource/resourcemanager.py [hunk 1]
```diff
@@ -48,7 +48,8 @@ def share(
             # Missing / invalid timeouts are validated by the client.
             # Don't pre-check it here and set the timeout to max int.
             timeout = client_options.timeout or sys.maxsize
-            now = get_timestamp_utc()
+            # Use system time like simple-transfer does
+            now = datetime.now().timestamp()
```
Comment (maaktweluit) [inline golem/resource/resourcemanager.py:52]: shouldnt this be `default_now` like in the database?

**Review by maaktweluit — APPROVED** [review]
LGTM!

One small comment and a q:
Shouldnt this be fixed in simple-transfer instead?

File: golem/resource/resourcemanager.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply golem/resource/resourcemanager.py:52]: `default_now` returns a UTC `datetime` object

**Comment by mfranciszkiewicz** [comment]
@maaktweluit We'd have to release a new version of simple-transfer and afaict that wasn't planned

**Pull request merged**